public/data/mainfunktion.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Sean-michael Chambers",
+  "type": "personal",
+  "bio": "Software engineering student currently learning python and solving kata's on codewars.",
```

What do you think if we capitalize `python` and `codewars` here?
```suggestion
  "bio": "Software engineering student currently learning Python and solving kata's on Codewars.",
```